src/boot/efi/boot.c

```diff
@@ -1257,6 +1273,20 @@ static void config_defaults_load_from_file(Config *config, char *content) {
                         continue;
                 }
 
+                if (streq8(key, "auto-shutdown")) {
+                        err = parse_boolean(value, &config->auto_shutdown);
+                        if (err != EFI_SUCCESS)
+                                log_error("Error parsing 'auto-shutdown' config option: %s", value);
```

Comment:
in userspace we typically suffix such messages with ", ignoring", to indicate that we don't consider this fatal for anything. i.e.

```c
log_error("Error parsing 'auto-shutdown' config option, ignoring: %s", value);
```

Now, I understand you just copied this from above, but let's maybe pick a wording for new stuff that is more in line with how we do things today.